deno.jsonc

```
@@ -0,0 +1,10 @@
+{
+  "lock": false,
+  "tasks": {
+    "check": "deno check --no-lock denops/**/*.ts",
```

`--no-lock` isn't needed since `lock` is set in the config file.

```suggestion
    "check": "deno check denops/**/*.ts",
```